roles/addons/diskless/files/livenet_module.py

```diff
@@ -139,58 +139,56 @@ def generate_file_system(self):
     def generate_operating_system(self):
         logging.info('Generating image \'' + self.name + '\' operating system')
 
-        # create image personnal working directory
+        # Create os generation directory
         logging.debug('Executing \'mkdir -p ' + self.WORKING_DIRECTORY + 'generated_os\'')
         os.makedirs(self.WORKING_DIRECTORY + 'generated_os')
 
+        # Set up release version
         if hasattr(self, 'release_version'):
-            release = '--releasever=' + self.release_version
+            release = ' --releasever=' + self.release_version
         else:
             release = ''
 
-        # Populating dnf packages list
-        dnf_packages_list = []
+        # Create empty string of packages
+        dnf_packages = ''
 
-        # Generate desired image file system
+        # Get appropriate packages for desired image file system
         if self.livenet_type == LivenetImage.Type.STANDARD:
             logging.debug('Standard image requested. Adding "@core" to packages list')
-            dnf_packages_list.append('@core')
+            dnf_packages = '@core '
+
         elif self.livenet_type == LivenetImage.Type.SMALL:
             logging.debug('Small image requested. Adding "dnf yum iproute procps-ng openssh-server NetworkManager" to packages list')
-            dnf_packages_list.append('dnf yum iproute procps-ng openssh-server NetworkManager')
+            dnf_packages = 'dnf yum iproute procps-ng openssh-server NetworkManager '
```

Comment:
Remove end blank space here. Add a blank at the beginning when you concatenate a string.